browser/ui/views/sidebar/sidebar_container_view.cc

```diff
@@ -36,10 +49,21 @@ void SidebarContainerView::Init() {
   UpdateChildViewVisibility();
 }
 
-void SidebarContainerView::ShowSidebar(bool show) {
-  sidebar_control_view_->SetVisible(show);
+void SidebarContainerView::SetSidebarShowOption(int show_option) {
```

Comment:
Shouldn't this specify an `enum` instead of an `int`? Or are we unable to expose that enum here?

Reply:
Thanks for catching this!.
Converted ShowSidebarOption as enum class and removed implicit converting to int.